src/models/v2/asyncapi.ts

```diff
@@ -17,4 +19,8 @@ export class AsyncAPIDocument
   info(): InfoInterface {
     return new Info(this._json.info);
   }
+
+  servers(): ServersInterface {
+    return new Servers(createArrayFromMap(this._json.servers));
+  }
```

Comment:
You are not creating a valid `Servers` instance. This class accepts `Server` instance arrays, not an array of normal objects:

```suggestion
  servers(): ServersInterface {
    return new Servers(
      Object.entries(this._json.servers).map(([serverName, server]) => new Server(serverName, server))
    );
  }
```

You have to import `Server` class.